src/components/network/network-map.js

```diff
@@ -94,10 +94,17 @@ const NetworkMap = (props) => {
 
     const readyToDisplayLines =
         readyToDisplay &&
-        props.mapEquipments.lines &&
+        (props.mapEquipments?.lines || props.mapEquipments?.hvdcLines) &&
```

Comment:
I'm not sure of this condition. hvdcLines may be downloaded quicker. Do we want to make a deck.gl render with only hvdc lines ?